python/test/unit/mesh/test_mixed_topology.py

```diff
@@ -0,0 +1,101 @@
+from mpi4py import MPI
+from dolfinx.cpp.mesh import create_topology, CellType
+import numpy as np
+
+
+def test_triquad():
+    cells = [[0, 1, 2, 1, 2, 3], [2, 3, 4, 5]]
+    orig_index = [[0, 1], [2]]
+    ghost_owners = [[], []]
+    boundary_vertices = []
+
+    topology = create_topology(MPI.COMM_SELF, [CellType.triangle, CellType.quadrilateral],
+                               cells, orig_index, ghost_owners, boundary_vertices)
+
+    maps = topology.index_maps(topology.dim)
+    assert len(maps) == 2
+    # Two triangles and one quad
+    assert maps[0].size_local == 2
+    assert maps[1].size_local == 1
+
+    # Six vertices in map
+    map0 = topology.index_maps(0)
```

Comment:
Should `index_maps` rather be a property that return a set?

Reply:
It would then match `entity_types`.

Reply:
It could be. Or we could use `ìndex_map(dim, index)`. I guess I was keeping it the same as the single-cell-type versions.

Reply:
I'll keep it for now